pkg/runtime/plan/dal/show_table_status.go

```diff
@@ -112,6 +113,18 @@ func (s *ShowTableStatusPlan) ExecIn(ctx context.Context, conn proto.VConn) (pro
 		}
 		sm.Store(dest[0], "")
 		return true
+	}), dataset.Filter(func(next proto.Row) bool {
+		var vr rows.VirtualRow
+		switch val := next.(type) {
+		case mysql.TextRow, mysql.BinaryRow:
+			return true
+		case rows.VirtualRow:
+			vr = val
+		default:
+			return true
+		}
+		tableName := vr.Values()[0].(string)
+		return strings.HasPrefix(tableName, aranaSystemTablePrefix)
```

Comment:
`aranaSystemTablePrefix`  is  a constant at the `show_tables` file, suggest define it to a common file.